backend/data/dataSources.py

```diff
@@ -262,37 +222,22 @@
 
 # Boundary files to assimilate data into (as 'properties')
 BOUNDARYFILES = {
-    "LA": {
-        "path": "data/static/geoboundaries/boundaries_LAs.geojson",
-        "ID_name": "lad18cd",
-        "area_name": "lad18nm",
-    },
-    "LSOA": {
-        "path": "data/static/geoboundaries/boundaries_LSOAs.geojson",
-        "ID_name": "LSOA11CD",
-        "area_name": "LSOA11NM",
-    },
-    "LHB": {
-        "path": "data/static/geoboundaries/boundaries_LHBs.geojson",
-        "ID_name": "lhb19cd",
-        "area_name": "lhb19nm",
-    },
-    "wales": {
-        "path": "data/static/geoboundaries/boundaries_Wales.geojson",
-        "ID_name": "ctry19cd",
-        "area_name": "ctry19nm",
-    },
+    "LA": {"path": "backend/data/geoboundaries/boundaries_LA.geojson", "ID_name": "lad18cd", "area_name":"lad18nm"},
+    "LSOA": {"path": "backend/data/geoboundaries/boundaries_LSOA.geojson", "ID_name": "LSOA11CD", "area_name":"LSOA11NM"},
```

Comment:
as above

Reply:
Yes well this makes more sense, there are only 4 entries and these are qualitative fields so I will actually make your suggestion as a change :)

Reply:
Multi-line does help with comprehension so it's not about aesthetics but it doesn't matter because valerio is going to change this into data classes anyway